pom.xml

```diff
@@ -26,7 +26,7 @@
         <!-- Lib versions - keep list sorted on property name -->
         <geotools.version>21.2</geotools.version>
         <geotools.wfs.version>16.5</geotools.wfs.version>
-        <jackson.version>2.10.1</jackson.version>
+        <jackson.version>2.12.3</jackson.version>
```

Comment:
Sorry for being a little over-conservative, but was there a specific feature that you want from 2.12.3 or did you just want to upgrade to the latest version?

Reply:
Hi! Appreciate your help, sounds great--this change was made in reference to one of GitHub's dependabots flagging the old version as having a security issue I believe. It's not required, I should have probably removed even.

Reply:
Okay, I removed this. Obviously you should consider updating separately given the security issue.